### PR TITLE
Added Analog Clock (SWoC)

### PR DESCRIPTION
**Pull request by DragonUncaged**
#18 
Demo
https://webdevtest.dragonuncaged.repl.co/

image

### CODE REVIEW

File: Analog_Clock/style.css [hunk 1]
```diff
@@ -0,0 +1,198 @@
+body {
```
Comment (Ovenoboyo) [inline Analog_Clock/style.css:1]: Avoid styling html tags. Instead use css classes wherever needed

File: Analog_Clock/index.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            <p style="color:white">8</p>
+            </span>
+        </li>
+        <li>
+            <span>
+            <p style="color:green">9</p>
+            </span>
+        </li>
+        <li>
+    <!--         <span> -->
```
Comment (Ovenoboyo) [inline Analog_Clock/index.html:59]: Unnecessary comments. It would be better to remove them

**Review by Ovenoboyo — CHANGES_REQUESTED** [review]

**Comment by DragonUncaged** [comment]
Made All the necessary Changes.
Re-review.

**Review by thepranaygupta — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]

File: Analog_Clock/style.css [hunk 3]
```diff
@@ -121,7 +128,6 @@ body {
     margin: 100px auto;
   }
   
-  /* JavaScript pre-rotates these wrappers instead of the hands when getting the time at load. If it pre-rotates the hands, then the hands will not perform 1 turn at their normal speeds. */
```
Comment (Ovenoboyo) [inline Analog_Clock/style.css]: this was a nice comment lol, wasn't unnecessary. But its ok

**Review by Ovenoboyo — APPROVED** [review]

File: Analog_Clock/style.css (hunk 3 above)
Comment (DragonUncaged) [reply Analog_Clock/style.css]: So My Pull Request will get Accepted !

Comment (Ovenoboyo) [reply Analog_Clock/style.css]: Yes its approved from my side. It will be merged by the mentor of this repo when everything is sorted from their side.

**Review by Abirpal202049 — APPROVED** [review]

**Pull request merged**